kuma/search/filters.py

```diff
@@ -184,7 +188,8 @@ def filter_queryset(self, request, queryset, view):
             if len(filter_tags) > 1:
                 facet_params = F('terms', tags=list(filter_tags))
             else:
-                facet_params = F('term', tags=filter_tags[0])
+                if filter_tags:
+                    facet_params = F('term', tags=filter_tags[0])
```

Comment:
Okay, I've made this change as requested. Feel free to give this a try and see if it works as intended. @jwhitlock @stephaniehobson